django/db/migrations/serializer.py

```diff
@@ -125,8 +125,14 @@ class EnumSerializer(BaseSerializer):
     def serialize(self):
         enum_class = self.value.__class__
         module = enum_class.__module__
+        if issubclass(enum_class, enum.Flag):
+            members, __ = enum._decompose(enum_class, self.value)
+        else:
+            members = (self.value,)
```

Comment:
In Python 3.11 it's possible to iterate over members instead of using a private API. Moreover `_decompose` is not available anymore (see https://github.com/python/cpython/commit/7aaeb2a3d682ecba125c33511e4b4796021d2f82):

```suggestion
        if issubclass(enum_class, enum.Flag):
            if PY311:
                members = list(self.value)
            else:
                members, _ = enum._decompose(enum_class, self.value)
                members = reversed(members)
        else:
            members = (self.value,)
```

Reply:
`reversed` to get members in definition order, as in Python 3.11.